googleapis/cloud/asset/v1/asset_service.pb.go

```diff
@@ -774,6 +1400,96 @@ func _AssetService_BatchGetAssetsHistory_Handler(srv interface{}, ctx context.Co
 	return interceptor(ctx, in, info, handler)
 }
 
+func _AssetService_CreateFeed_Handler(srv interface{}, ctx context.Context, dec func(interface{}) error, interceptor grpc.UnaryServerInterceptor) (interface{}, error) {
```

Comment:
@noahdietz what's up with these underscores? o.O

Reply:
I have no clue. But they've been in here since the file was added....must be how gRPC hides these handlers from the pkg API.

Reply:
yeah, must be to unexport those symbols. kinda funky!